### ISSUE TITLE
Thank you for the mod, Chinese simplified translation completed

### ISSUE THREAD

**Issue by Kirisoup** [alejandrocoria/MapFrontiers#18]
[zh_cn.zip](https://github.com/alejandrocoria/MapFrontiers/files/9493025/zh_cn.zip)

**Comment by alejandrocoria** [comment]
Hello, I modified a few lines for the new version. Can you translate them please? (line 74)

[zh_cn.zip](https://github.com/alejandrocoria/MapFrontiers/files/9594202/zh_cn.zip)

**Comment by dlfnn** [comment]
[zh_cn.zip](https://github.com/alejandrocoria/MapFrontiers/files/9596136/zh_cn.zip)
no problem!

**Comment by alejandrocoria** [comment]
Thank you very much! In one or two days I upload the new version.

**Comment by alejandrocoria** [comment]
Added in 2.4.0-beta1.

**Comment by HeyLoon** [comment]
> [zh_cn.zip](https://github.com/alejandrocoria/MapFrontiers/files/9493025/zh_cn.zip)

hey Kirisoup,
Can I refer to this document for Traditional Chinese (Hongkong) and Traditional Chinese (Taiwan) translation?

**Comment by Kirisoup** [comment]
> > [zh_cn.zip](https://github.com/alejandrocoria/MapFrontiers/files/9493025/zh_cn.zip)
> 
> 
> 
> hey Kirisoup,
> 
> Can I refer to this document for Traditional Chinese (Hongkong) and Traditional Chinese (Taiwan) translation?

Of course! As long as it helps, please go ahead and do so. But one thing that I'm afraid is that my translation might be somewhat outdated, which maybe needs to be considered.

**Comment by alejandrocoria** [comment]
This is the file that is currently used in 1.20.1:
https://github.com/alejandrocoria/MapFrontiers/blob/4bb718966c0e9fbc423410b0c7fac3c41db84682/Common/src/main/resources/assets/mapfrontiers/lang/zh_cn.json
As you can see, it is a bit outdated.
